IDE/MQX/Makefile

```diff
@@ -0,0 +1,153 @@
+WOLF_ROOT = ../..
+MQX_ROOT = /mnt/c/Freescale/Freescale_MQX_4_1
```

Comment:
Can you setup a relative path by default? Can you add instructions in the README for where to get MQX and the GCC ARM compiler used? Also add a comment about what this generates (static wolfSSL library). I don't think this will actually build correctly without specifying `MQXLIB`... can you document that and how to get/build?

Reply:
fixed